.github/workflows/tests.yml

```
@@ -62,7 +62,7 @@ jobs:
         if: ${{ matrix.os == 'macos-12' }}
         run: colima start
 
-      - uses: extractions/setup-just@95b912dc5d3ed106a72907f2f9b91e76d60bdb76  # 1.4
+      - uses: extractions/setup-just@69d82fb0233557aec017ef13706851d0694e0f1d  # 1.4
```

```suggestion
      uses: extractions/setup-just@69d82fb0233557aec017ef13706851d0694e0f1d
```
version number does not match hash